metadata-builders/src/main/java/com/linkedin/metadata/builders/search/RegisteredIndexBuilders.java

```diff
@@ -17,6 +17,7 @@
         {
           add(new CorpUserInfoIndexBuilder());
           add(new DatasetIndexBuilder());
+          add(new DataProcessIndexBuilder());
```

Comment:
In general we like to keep the list alphabetically (and case insensitively) ordered so swap this with line 19?